include/oneapi/dpl/experimental/kt/esimd_radix_sort.h

```diff
@@ -25,44 +25,32 @@
 
 namespace oneapi::dpl::experimental::kt::esimd::impl
 {
-template <typename T, ::std::enable_if_t<sizeof(T) <= sizeof(::std::uint32_t), int> = 0>
-constexpr ::std::uint32_t
-__onesweep_process_size()
-{
-    return 384;
-}
-
-template <typename T, ::std::enable_if_t<sizeof(T) == sizeof(::std::uint64_t), int> = 0>
-constexpr ::std::uint32_t
-__onesweep_process_size()
-{
-    return 192;
-}
 
 // TODO: allow calling it only for all_view (accessor) and guard_view (USM) ranges, views::subrange and sycl_iterator
-template <bool _IsAscending, std::uint32_t _RadixBits, typename _KernelParam, typename _Range>
+template <bool _IsAscending, std::uint16_t _RadixBits, typename _KernelParam, typename _Range>
```

Comment:
Why ```std::uint16_t _RadixBits``` ? I believe we may use ```std::uint8_t``` type too.

Reply:
It looks reasonable, because the supported sizes are arithmetic ones.